src/Administration/Resources/app/administration/src/core/service/login.service.ts

```diff
@@ -22,6 +22,20 @@ interface TokenResponse {
     /* eslint-enable camelcase */
 }
 
+interface SsoProvider {
+    type: string;
+    snippetKey: string;
+    icon: string;
+    class: string;
+    url: string;
+    additionalData: [];
```

Comment:
Do we really want `additionalData` in the frontend? I'm a little worried that it might leak secrets if some provider implementation puts some additional credentials in there...